ca/django_ca/models.py

```diff
@@ -793,11 +793,6 @@ def key(self, password: Optional[Union[str, bytes]]) -> PrivateKeyTypes:
 
             self._key = load_pem_private_key(key_data, password, default_backend())
 
-        # type checks only exist to make mypy happy
-        if isinstance(self._key, ed25519.Ed25519PrivateKey):  # pragma: nocover
-            raise ValueError("Ed25519 private keys are not supported.")
-        if isinstance(self._key, ed448.Ed448PrivateKey):  # pragma: nocover
-            raise ValueError("Ed25519 private keys are not supported.")
```

Comment:
I think you should not remove the check for `ed448` keys, right?

Note that mypy checks are currently not in the GitHub CI pipeline, as it only works with the current HEAD of cryptography (in fact, I made quite a few PRs for this in cryptography). But there is a tox target that you can run locally for MyPy checks!

Reply:
Restored ed448 check